api-client/src/endpoints/__tests__/createCitiesEndpoint.spec.ts

```diff
@@ -74,7 +74,7 @@ describe('cities', () => {
             latitude: 48.3705449
           }
         },
-        boundingBox: [10.7880103, 48.447238, 11.0174493, 48.297834]
+        boundingBox: [48.447238, 10.7880103, 48.297834, 11.0174493]
```

Comment:
I think it would make sense to change it in the django endpoint, as you normally always start with the latitude, when writing coordinates. https://stackoverflow.com/questions/7309121/preferred-order-of-writing-latitude-longitude-tuples-in-gis-services

Reply:
since everything is now done in cords[long,lat] we can also keep it. there is no clear rule for that. the documentations differ. Check my comment for mapbox f.e. i don't see that big value in changing everything. We just have to keep it everything there same